src/zenml/cli/model.py

```diff
@@ -515,7 +521,7 @@ def _print_artifacts_links_generic(
     help="List artifacts linked to a model version.",
 )
 @click.argument("model_name_or_id")
-@click.argument("model_version_name_or_number_or_id")
+@click.argument("model_version_name_or_number_or_id", default="0")
```

Comment:
This feels a bit weird, why can't we use `None` as default / latest value instead?

Reply:
Cause it is strings - cli arguments and `-1` which I planned to use is treated as a parameter name, so 0 seemed a solid choice. Moreover - user will not specify it directly, it will be set by just skipping this arg.
`zenml model version artifacts my_model` -> will return artifacts for latest in `my_model`

Reply:
I don't get it, why can't it be `Optional[str]` instead? I.e., why would this not work?

```python
@click.argument("model_name_or_id")
@click.argument("model_version_name_or_number_or_id")
def list_model_version_artifacts(
    model_name_or_id: str,
    model_version_name_or_number_or_id: Optional[str] = None,
) -> None:
    ...

def _print_artifacts_links_generic(
    model_name_or_id: str,
    model_version_name_or_number_or_id: Optional[str] = None,
    ...
):
    ...
```

Reply:
It is a click limitation, as I see this. If I set `default=None` it is treated as no default, making arg mandatory. Changes on the level of functions args are not effective at all - `@click.argument` is the king here.

Reply:
Ah, yes, `click.argument` means mandatory, for optional inputs you need to use `click.option`